### ISSUE TITLE
Bug: Adding fill to certain Graphics with reversed normals causes lineStyle stroke to reverse inner/outer position

### ISSUE THREAD

**Issue by joshstrike** [pixijs/pixijs#10169]
### Current Behavior

Certain shapes drawn with quadratic curves draw an "inside" or "outside" stroke opposite to their expected position depending on whether you've called `beginFill()` on the graphic.

While what constitutes inner or outer stroke around a graphic may be expected to change depending on the alignment of the path normals, "inner" or "outer" should remain consistent for any given shape and not vary or flip depending on whether it's filled.

### Expected Behavior

`lineStyle(10,0x000,1,1)` should produce whatever an "outer" line would be given the path, regardless of whether the shape is filled.

### Steps to Reproduce

Punch this into https://pixijs.com/playground/ and try changing the argument for `testInnerOuterStroke(n)` from 1 to 0. Both shapes are drawn exactly the same except for whether or not they're filled:

```
import * as PIXI from 'pixi.js';

const app = new PIXI.Application({ background: '#1099bb', resizeTo: window });

document.body.appendChild(app.view);

//CHANGING THIS TO 0 causes the stroke on both to be "inner"
//THE ONLY DIFFERENCE BETWEEN THE TWO IS THAT ONE HAS A FILL.
//If normal flipping was not contingent on fill, they should 
//appear the same in both cases.
testInnerOuterStroke(1);

function testInnerOuterStroke(_innerOuterStroke=0) {
    const a = new PIXI.Graphics();
    a.lineStyle(10,0x000,1,   _innerOuterStroke);
    a.quadraticCurveTo(-214,207,-96,344);
    a.quadraticCurveTo(-194,207,76,344);
    a.x = 200;
    app.stage.addChild(a);

    const b = new PIXI.Graphics();
    b.beginFill(0xFF0000);
    b.lineStyle(10,0x000,1,   _innerOuterStroke);
    b.quadraticCurveTo(-214,207,-96,344);
    b.quadraticCurveTo(-194,207,76,344);
    b.x = 400;
    app.stage.addChild(b);
}
```

### Environment

- **`pixi.js` version**: *e.g. 7.1.0*
- **Browser & Version**: *e.g. Chrome 108*
- **OS & Version**: *e.g. Ubuntu 22.04*
- **Running Example**: *e.g. <https://pixiplayground.com/>*

**Comment by Zyie** [comment]
Thanks for reporting this!

As this issue relates to v6, and given our limited resources, we're currently focusing our efforts on maintaining and advancing v8. Because of that, we’re going to close this issue for now.

That said, if you find this issue still applies to v8 , please feel free to open a new issue and we’ll be happy to take a look.
